apps/aeutils/src/aeu_mp_trees.erl

```diff
@@ -658,7 +658,7 @@ hexchar(X) when X > -1, X < 16 -> $A + X - 10.
 
 -spec node_hash(enc_node()) -> hash().
 node_hash(<<>>) -> error(no_hash_for_null);
-node_hash(Bin) when byte_size(Bin) < 32   -> crypto:hash(sha256, Bin);
+node_hash(Bin) when byte_size(Bin) < 32   -> aec_hash:hash(header, Bin);
```

Comment:
I see this line has assumption that `aec_hash:hash(header, ...)` is 32 bytes - that is valid. If implementation of aec_hash changes I expect tests of aeu_mp_trees to fail (on decode) so no change needed.